reference_docs/YAML_models/ietf_v2_document_attributes.yaml

```diff
@@ -0,0 +1,524 @@
+version: v3.4.5 #Verify versions
+
+inherits_from:
+  # Also may be referred to as “parent flavor”
+  flavor: standoc
+  version: v1.x.x
+
+document_attributes:
+#--- Document type  ---#
+  doctype:
+    tags: ["Document type"]
+    title: "Document type"
+    description: |
+      "Specifies document status. Choices:
+
+      * `:doctype: internet-draft` sets the document as an Internet-Draft (default value):
+      ** `rfc@docName` will be set to the `:name:` attribute
+      * `:doctype: rfc` sets the document as an RFC:
+      ** `rfc@number` will be set to the `:name:` attribute".
+    value_default: internet-draft
+
+#--- Global options  ---#
+  no-rfc-bold-bcp14:
+    tags: ["Global options"]
+    title: "No RFC bold BCP14"
+    description: "Override default assumption that boldface uppercase BCP14 word is to be rendered with `bcp14` tag.".
+    value_spec: boolean
+    value_default: true
+
+  smart-quotes:
+    tags: ["Global options"]
+    title: "Smart quotes"
+    description: |
+      "Permit smart quotes, when they are specified explicitly in AsciiDoc (as `pass:['`...`']`, `pass:['`...`']`). When disabled, smart quotes are rendered as straight quotes, and Asciidoc's default conversion of straight apostrophes to smart is undone.".
+    value_spec: boolean
+    value_default: true
+
+  inline-definition-lists:
+    tags: ["Global options"]
+    title: "Inline definition lists"
+    description: |
+      'Only applies to v2. By default, `<vspace blankLines="1">` is inserted after the definition in a v2 definition list, to satisfy the requirement from `idnits` validation that definition terms be separated by a carriage return from the definition. That is, by default inline definition lists are rendered as paragraphing definition lists. If the option is on, the additional `vspace` element is not added, and inline definition lists are left as is.'.
+    value_spec: boolean
+    value_default: false
+
+  flush-caches:
```

Comment:
v2 is incompatible with the rest of Metanorma. This does override, but I don't want us to talk about v2 really anyway, it is a legacy format. (And I greatly doubt anyone is using it.)

Reply:
Do we need to maintain v2 docs if we can verify that all of IETF moved to v3?

Reply:
FYI: IETF I'm sure have not moved to v3 completely, because v3 remains unstable five years on (sigh), and I doubt that our early v2 parser ever got used. But we did lodge an RFC describing that parser, which I think obligates us to maintain documentation for it, as legacy only.